src/pages/account-menu/in-an-app-bar/index.tsx

```diff
@@ -0,0 +1,348 @@
+                                </Typography>
+                                <Typography variant={'body1'} noWrap>
+                                    Showing A Custom Profile Picture
+                                </Typography>
+                            </div>
+                            <Spacer />
+                            <UserMenu
+                                avatar={<Avatar alt="Chima Thabani" src={avatarImage} />}
+                                menuGroups={menuGroupItems}
+                                menuTitle={avatarTitle}
+                                menuSubtitle={avatarSubtitile}
+                                MenuProps={{
+                                    anchorOrigin: {
+                                        vertical: 'bottom',
+                                        horizontal: 'right',
+                                    },
+                                    transformOrigin: {
+                                        vertical: 'top',
+                                        horizontal: 'right',
+                                    },
+                                    classes: { paper: classes.paper },
+                                }}
+                            />
+                        </Toolbar>
+                    </AppBar>
+                    {/* Status Avatar Example */}
+                    <AppBar position="static" color="inherit" className={classes.appBar}>
+                        <Toolbar classes={{ gutters: classes.toolbarGutters }}>
+                            <IconButton edge={'start'}>
+                                <MenuIcon />
+                            </IconButton>
+                            <div className={classes.textContainer}>
+                                <Typography variant={'h6'} noWrap>
+                                    Status Avatar
+                                </Typography>
+                                <Typography variant={'body1'} noWrap>
+                                    Avatar with Status Indicator
+                                </Typography>
+                            </div>
+                            <Spacer />
+                            <UserMenu
+                                avatar={
+                                    <Badge
+                                        overlap="circular"
+                                        anchorOrigin={{
+                                            vertical: 'bottom',
+                                            horizontal: 'right',
+                                        }}
+                                        variant="dot"
+                                        classes={{ dot: classes.badge }}
+                                    >
+                                        {<Avatar alt="Chima Thabani" src={avatarImage} />}
+                                    </Badge>
+                                }
+                                menuGroups={[
```

Comment:
can we use the same menuGroupItems object that the others are sharing?

Reply:
Sure, @huayunh shall I make common menuGroupItems for all variation?

Reply:
One of the menus is different (the anonymous generic person avatar). The others should be the same. In one of your implementations, they are not using the shared definition (I was able to tell because it says "Logout" instead of "Log Out")